src/os/data/columndefinition.js

```diff
@@ -292,6 +292,9 @@ os.data.ColumnDefinition.prototype.restore = function(config) {
   if (config.hasOwnProperty('selectable')) {
     this['selectable'] = config['selectable'];
   }
+  if (config.hasOwnProperty('formatter')) {
```

Comment:
In one iteration of my code, I had needed it to quickly rebuild a typed ColumnDefinition object from a basic JS object.  Since then I removed that code and left this change since the restore() function didn't appear to be called by anything that would have a "formatter" on it.

For safety, I'll remove this change -- especially since I no longer need it.